nav2_costmap_2d/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ find_package(tf2_ros REQUIRED)
 find_package(tf2_sensor_msgs REQUIRED)
 find_package(visualization_msgs REQUIRED)
 find_package(angles REQUIRED)
+find_package(grid_map_core REQUIRED)
```

Comment:
This would be a disqualifying requirement on the contribution if it is needed. We aren't going to introduce grid maps as a requirement to Costmap 2D